storage/range.go

```diff
@@ -192,28 +192,13 @@ func (r *Range) stop() {
 
 // Destroy cleans up all data associated with this range.
 func (r *Range) Destroy() error {
```

Comment:
On the other hand, column families make reads and writes slower if access is uniformly distributed across the families (and I'm not sure how well it would cope with frequent creation and removal of column families; this isn't really the use case it was designed for).